test/performance/specs/site-editor.spec.js

```diff
@@ -438,9 +438,7 @@ test.describe( 'Site Editor Performance', () => {
 				await Promise.all(
 					Array.from( { length: perPage }, async ( el, index ) => {
 						return await page
-							.getByRole( 'link', {
```

Comment:
This test assumed the page was using the `table` layout, which was the active one when the Trash view was loaded — and remained so. This PR changes that behavior and always loads the layout associated with the default view, hence at this point the layout will be list.

The test failed because the Pages are links in the table layout but they are not in the list layout.